README.md

```diff
@@ -1 +1,7 @@
-# superx
+superx
+Team members:
```

Comment:
How is this change related to the PR description. Do NOT mix few efforts in one PR.